docs/flink-connector/flink-catalog-paimon.md

```diff
@@ -0,0 +1,114 @@
+
+show databases;
+-- +---------------------+
+-- |       database name |
+-- +---------------------+
+-- |             default |
+-- | gravitino_paimon_db |
+-- +---------------------+
+
+SET 'execution.runtime-mode' = 'batch';
+-- [INFO] Execute statement succeed.
+
+SET 'sql-client.execution.result-mode' = 'tableau';
+-- [INFO] Execute statement succeed.
+
+CREATE TABLE paimon_tabla_a (
+    aa BIGINT,
+    bb BIGINT
+);
+
+show tables;
+-- +----------------+
+-- |     table name |
+-- +----------------+
+-- | paimon_table_a |
+-- +----------------+
+
+
+select * from paimon_table_a;
+-- Empty set
+
+insert into paimon_table_a(aa,bb) values(1,2);
+-- [INFO] Submitting SQL update statement to the cluster...
+-- [INFO] SQL update statement has been successfully submitted to the cluster:
+-- Job ID: 74c0c678124f7b452daf08c399d0fee2
+
+select * from paimon_table_a;
+-- +----+----+
+-- | aa | bb |
+-- +----+----+
+-- |  1 |  2 |
+-- +----+----+
+-- 1 row in set
+```
+
+## Catalog properties
+
+Gravitino Flink connector will transform below property names which are defined in catalog properties to Flink Paimon connector configuration.
+
+| Gravitino catalog property name | Flink Paimon connector configuration   | Description                                                                                                                                                                                                 | Since Version    |
+|---------------------------------|----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------|
+| `catalog-backend`               | `metastore`                            | Catalog backend of Gravitino Paimon catalog. Supports `filesystem`, `jdbc` and `hive`.                                                                                                                      | 0.8.0-incubating |
+| `uri`                           | `uri`                                  | The URI configuration of the Paimon catalog. `thrift://127.0.0.1:9083` or `jdbc:postgresql://127.0.0.1:5432/db_name` or `jdbc:mysql://127.0.0.1:3306/metastore_db`. It is optional for `FilesystemCatalog`. | 0.8.0-incubating |
+| `warehouse`                     | `warehouse`                            | Warehouse directory of catalog. `file:///user/hive/warehouse-paimon/` for local fs, `hdfs://namespace/hdfs/path` for HDFS , `s3://{bucket-name}/path/` for S3 or `oss://{bucket-name}/path` for Aliyun OSS  | 0.8.0-incubating |
+| `oss-endpoint`                  | `fs.oss.endpoint`                      | The endpoint of the Aliyun OSS.                                                                                                                                                                             | 0.8.0-incubating |
```

Comment:
could you remove cloud storage related configuration, since this is not tested yet.